public/pages/Indices/components/ApplyPolicyModal/ApplyPolicyModal.tsx

```diff
@@ -283,7 +283,13 @@ export default class ApplyPolicyModal extends Component<ApplyPolicyModalProps, A
             // @ts-ignore */}
         <EuiModal onCancel={onClose} onClose={onClose}>
           <EuiModalHeader>
-            <EuiModalHeaderTitle>Apply policy</EuiModalHeaderTitle>
+            <EuiModalHeaderTitle>
+              {" "}
+              <EuiText size="s">
+                {" "}
+                <h2>Apply policy</h2>{" "}
+              </EuiText>{" "}
+            </EuiModalHeaderTitle>
           </EuiModalHeader>
 
           <EuiModalBody>
```

Comment:
Not sure if the text is intentionally xs, should the apply policy paragraph text be wrapped in `EuiText size="s"` instead?
<img width="598" alt="Screenshot 2024-08-29 at 11 18 22 AM" src="https://github.com/user-attachments/assets/6743c9c6-d772-4b5c-9125-1f8ccdf9e832">